POLYFILLS.md

```diff
@@ -6,20 +6,50 @@ browsers yet. In order to maximize browser compatibility, you should install
 [polyfills](https://en.wikipedia.org/wiki/Polyfill_(programming)) to fill in the
 gaps for some of the newest features.
 
+---
+
+ - ✅ Natively supported
+ - 🚧 Available with polyfill
+ - 🚫 Not available
+ - 🎌 Behind a flag, unstable
+
+Feature                   | Chrome | Canary | Safari 12 | Firefox 64 | Firefox 62 | Edge  | IE 11
+--------------------------|--------|--------|-----------|------------|------------|-------|------
+Resize Observer¹          |     ✅ |     ✅ |        🚧 |         🚧 |         🚧 |    🚧 |   🚧
+Custom Elements           |     ✅ |     ✅ |        ✅ |         🚧 |         🚧 |    🚧 |   🚧
+Shadow DOM                |     ✅ |     ✅ |        ✅ |         ✅ |         🚧 |    🚧 |   🚧
+Intersection Observer²    |     ✅ |     ✅ |        🚧 |         ✅ |         ✅ |    ✅ |   🚧
+Fullscreen API³           |     ✅ |     ✅ |        🚧 |         ✅ |         🚧 |    🚧 |   🚧
+WebXR Device API          |     🚫 |     🎌 |        🚫 |         🚫 |         🚫 |    🚫 |   🚫
+WebXR HitTest API         |     🚫 |     🎌 |        🚫 |         🚫 |         🚫 |    🚫 |   🚫
+
+_1: Resize Observer is optional. [See below](https://github.com/PolymerLabs/model-viewer/blob/master/POLYFILLS.md#regarding-resize-observer) for more details_
```

Comment:
Maybe put CE/Shadow DOM up top as they're required, and others are enhancements

Reply:
The below sections clarify this a bit more, this is related to the initial-glance at the chart seeing if it's "ready", and depending on how you've heard of this (e.g. as an "augmented reality component"), you may assume that it might not work in all browsers

Reply:
Will fix.

Reply:
Or split it into a separate section entirely? That might help differentiate between what we need to work vs what makes the experience better.

Reply:
I suppose the organization that I'm envisioning is something like

```
# Required Platform Features
<chart w/ custom elements & shadow dom>

# Optional Platform Features
<chart w/ resize & intersection observers>

## Resize Observer

## Intersection Observer

# WebXR (Experimental Platform Features)
<chart with fullscreen & webxr>

more info 
```